esp/templates/inclusion/program/class_teacher_list_row.html

```diff
@@ -1,4 +1,32 @@
-{# expects program, class, and crmi (ClassRegModuleInfo) #}
+{# expects program, class, can_req_cancel, and crmi (ClassRegModuleInfo) #}
+
+<style>
```

Comment:
This should really go in a proper CSS file, especially since this template gets rendered many times so putting it here means we're adding a `<style>` for every class.  Not sure offhand what file is best but I'm guessing the page already has something like `esp/public/media/styles/catalog.css` and you can just add it there.